TAPRegExt-v1.1.xsd

```diff
@@ -602,4 +616,93 @@ xsi:schemaLocation="http://www.w3.org/2001/XMLSchema http://vo.ari.uni-heidelber
     </xs:simpleContent>
   </xs:complexType>
 
+	<xs:complexType name="PerModeLimits">
```

Comment:
There's one per-mode use case that I'm thinking about that doesn't logically fit within `PerModeLimits`: output formats with mode-specific technical constraints. For example, we're considering VOParquet support exclusively for async queries, since partitioning large result sets across multiple files isn't feasible within sync queries.

What would be the optimal design if we wan to consider this? Should we rename `PerModeLimits` to `PerModeCapabilities` to encompass both limits and format capabilities, or would per-mode formats warrant a separate grouping?